core/src/main/java/org/apache/accumulo/core/clientImpl/ClientConfConverter.java

```diff
@@ -166,19 +166,14 @@ public String get(Property property) {
         if (property.isSensitive()) {
           org.apache.hadoop.conf.Configuration hadoopConf = getHadoopConfiguration();
           if (hadoopConf != null) {
-            try {
-              char[] value = CredentialProviderFactoryShim
-                  .getValueFromCredentialProvider(hadoopConf, key);
-              if (value != null) {
-                log.trace("Loaded sensitive value for {} from CredentialProvider", key);
-                return new String(value);
-              } else {
-                log.trace("Tried to load sensitive value for {} from CredentialProvider, "
-                    + "but none was found", key);
-              }
-            } catch (IOException e) {
-              log.warn("Failed to extract sensitive property ({}) from Hadoop CredentialProvider,"
-                  + " falling back to base AccumuloConfiguration", key, e);
+            char[] value = CredentialProviderFactoryShim.getValueFromCredentialProvider(hadoopConf,
```

Comment:
I don't think this credential provider stuff is very robust, as is... I'd be okay with ripping it out (or deprecating, so we can rip it out later, in favor of a better implementation for configuration property source provider).

Reply:
I agree but it should probably be done in a separate issue